types/git-clone/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for git-clone 0.2
+// Project: https://github.com/jaz303/git-clone
+// Definitions by: Christoph Thiede <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { } from './promise';
```

Comment:
```suggestion
```
oh, it's async version of the same api. You've already used it in tests, so it will be published,
no need to empty imports